### DISCUSSION TITLE
The border disappears

### DISCUSSION THREAD

**Discussion by chengjiazhao** [jxlsteam/jxls#142] [Q&A] [answered]
The border disappears after the cells are merged

![image](https://user-images.githubusercontent.com/42095720/126599256-3984b0de-63d0-4aee-a1ea-153ea3eb91c2.png)

**Comment by SoltauFintel** [comment, marked as answer]
I don't use the merge cell feature.

I guess you have to find a solution yourself. First you can create the template from scratch using MS Excel. Then have a look at the merge command in the JXLS sources. Maybe you need a new JXLS "make cell borders command"?
